_data/apis.yml

```diff
@@ -473,14 +720,48 @@
 
 - name: TranslateMe
   id: translateme
+  company: translateme
   urls:
    - https://translateme.network
    - https://translateme.network/api/
   languages: [ [de, en], [es, en], [fr, en], [it, en], [nl, en], [pt, en], [ru, en], [tr, en], [zh, en], [zu, en] ]
   self-serve: true
 
+- name: T-tact-AN-ZIN
+  id: t-tact-an-zin
+  company: toin
+  urls:
+    - https://to-in.com/en/service/mt/an-zin
+  privacy_url: https://to-in.com/en/security
+  languages: [ [en, my], [en, pt], [en, ru], [en, th], [en, vi], [en, zh-tw], [en, zn-cn], [ja, de], [ja, en], [ja, es], [ja, fr], [ja, id], [ja, it], [ja, it], [ja, ko], [ja, my], [ja, pt], [ja, th], [ja, vi], [ja, zh-cn], [ja, zh-tw] ]
+  stock_api_domains:
+    - finance
+    - legal
+    - patents
+  self-serve: false
+
+- name: Ubiqus NMT
+  id: ubiqus
+  company: ubiqus
+  urls:
+    - https://www.ubiqus.com/
+    - https://documenter.getpostman.com/view/8281683/SVtSX9x2?version=latest
```

Comment:
I'd try to remove junk like this on the URL

Reply:
Do you mean deleting the last part and having just `https://documenter.getpostman.com/view/8281683/SVtSX9x2`?